gatsby-browser.js

```diff
@@ -176,6 +176,11 @@ export const onClientEntry = () => {
     updateURLAsPerUserLanguage()
 }
 
+export const shouldUpdateScroll = () => {
```

Comment:
this is gonna cause too many issue, we shouldn't be using this.

Reply:
qa is raising issue about scrolling in all of the new approach pages. [https://app.clickup.com/t/20696747/WEBREL-382](https://app.clickup.com/t/20696747/WEBREL-382)

Reply:
can we use it as temporary solution?

Reply:
This is removed I assume @hasan-deriv you can resolve the conversation thanks 🙏